src/component/header.js

```diff
@@ -0,0 +1,312 @@
+    return (
+      <Container id="header">
+        <LogoContainer >
+          <Themeline
+            horizontal
+            alignTop
+          />
+          <LogoTitle>mooniak</LogoTitle>
+          <LogoContent >
+            We are a multydeciplinary design, art direction and technology studio. We provide
+            communication solutions and experience across
+            languages and mediums using graphic design, typography, image-making and
+            digital technologies.
+          </LogoContent>
+          <Themeline
+            horizontal
+            alignBottom
+          />
+        </LogoContainer>
+        <MenuContainer >
+          <CircleBase />
+          <Overlay>
+            <MenuRow>
+              <Column >
+                <Themeline
+                  horizontal
+                  alignTop
+                />
+                <MenuItem>about us</MenuItem>
+              </Column>
+              <Column>
+                <Row>
+                  <MenuItem>colombore</MenuItem>
+                  <Themeline vertical />
+                </Row>
+              </Column>
+            </MenuRow>
+            <MenuRow>
+              <MenuButton
+                href="#services"
+                onClick={ () => navigationCheck('services') }
+              >
+                OUR SERVICES
+              </MenuButton>
+              <MenuButton>TELL US YOUR NEED</MenuButton>
+            </MenuRow>
+            <MenuRow>
+              <Column >
+                <MenuItem
+                  href="#projects"
+                  onClick={ () => navigationCheck('projects') }
+                >projects</MenuItem>
+                <Themeline
+                  horizontal
+                  alignBottom
+                />
+              </Column>
+              <Column>
+                <Row>
+                  <MenuItem>subscribe</MenuItem>
+                </Row>
+              </Column>
+            </MenuRow>
+          </Overlay>
+        </MenuContainer>
+        <PartnersContainer >
+          <Row>
+            <PartnerText>partners</PartnerText>
+            <Themeline
+              horizontal
+              alignTop
+            />
+          </Row>
+          <PartnerLogo src={ Sense } />
+          <PartnerLogo src={ Texual } />
+          <PartnerLogo src={ Scrible } />
+        </PartnersContainer>
+        <ContactContainer >
+          <ContactRow border="solid">
+            0112255000
+          </ContactRow>
+          <ContactRow border="solid">
+            hello@mooniak.com
+          </ContactRow>
+          <ContactRow >
+            MooniakHQ, 33/1,
+            Siriwardena Rd,
+            Dehiwala, Colombo,
+            Sri Lanka
+          </ContactRow>
+          <Row>
+            <Themeline
+              alignTop
+              alignBottom
+              horizontal
+            />
+          </Row>
+          <SocialMediaRow >
+            <SocialIcon
```

Comment:
This should be wrapped inside a link `a`. Do that at first even if you don't have the link because adding it separately can break the style. But at the end remember to remove all empty links before publishing because it can effect SEO badly.